compass-js-frontend/src/main/webapp/ui/templates/editor/viewport.html

```diff
@@ -2,12 +2,11 @@
 	<xml3d activeview="#defaultView" class="xml3d">
 		<defs id="mainDefs">
 		</defs>
-		<view id="defaultView" position="1 1.5 13"></view>
+		<view id="defaultView" position="1 1.5 13">
+			<group data-hook="defaultLight"></group>
```

Comment:
To be fair though, I don't think anything is supported below a `<view>`, at least officially. Need to check whether it (by random chance) works while merging.